SharedPackages/DataBrokerProtectionCore/Sources/DataBrokerProtectionCore/Operations/SubJob/BrokerProfileOptOutSubJob.swift

```diff
@@ -47,6 +54,83 @@ struct BrokerProfileOptOutSubJob {
                           brokerProfileQueryData: BrokerProfileQueryData,
                           showWebView: Bool,
                           shouldRunNextStep: @escaping () -> Bool) async throws -> Bool {
+        // 1, 2, 3, 4. Validate function parameters and extract ID's
+        guard let identifiers = try validateOptOutPreconditions(for: extractedProfile,
+                                                                brokerProfileQueryData: brokerProfileQueryData,
+                                                                database: dependencies.database) else {
+            return false
+        }
```

Comment:
Just a thought: Instead of `validateOptOutPreconditions(for:brokerProfileQueryData:database)`, how about making a failable initializer like `OptOutIdentifiers.init?(for:brokerProfileQueryData:database)`?